Resources/Prototypes/_DV/Entities/Mobs/NPCs/carp.yml

```
@@ -0,0 +1,49 @@
+- type: entity
+  parent: MobShark
+  id: MobSharkColored
+  components:
+  - type: Sprite
+    drawdepth: Mobs
+    sprite: _DV/Mobs/Aliens/Carps/uncoloredsharkminnow.rsi
+    layers:
+    - map: [ "enum.DamageStateVisualLayers.Base" ]
+      state: alive
+    - map: [ "enum.DamageStateVisualLayers.BaseUnshaded" ]
+      state: mouth
+      shader: unshaded
+  - type: RandomSprite
+    available:
+    - enum.DamageStateVisualLayers.Base:
+        alive: Rainbow
+      enum.DamageStateVisualLayers.BaseUnshaded:
+        mouth: ""
+
+- type: entity
+  name: Sharkminnow
+  id: MobSharkminnowDragon
+  suffix: DragonBrood
+  parent: MobSharkColored
+  components:
+    - type: GhostRole
+      allowMovement: true
+      allowSpeech: true
+      makeSentient: true
+      name: ghost-role-information-sentient-shark-name
+      description: ghost-role-information-sentient-shark-description
+      rules: ghost-role-information-space-dragon-summoned-carp-rules
+      mindRoles:
+      - MindRoleGhostRoleTeamAntagonistFlock
+    - type: GhostTakeoverAvailable
+    - type: HTN
+      rootTask:
+        task: DragonCarpCompound
+    - type: Flammable
+      damage:
+       types: {}
+    - type: Temperature
+      heatDamageThreshold: 1200
+    - type: MeleeWeapon
+      damage:
+        types:
+          Slash: 10
+          Piercing: 10
```

this is all indented 2 spaces too much, the `-` should be right under the `c` of components

---

Its still valid YAML, but if they wanna change it, here's the suggestion.
```suggestion
  components:
  - type: GhostRole
    allowMovement: true
    allowSpeech: true
    makeSentient: true
    name: ghost-role-information-sentient-shark-name
    description: ghost-role-information-sentient-shark-description
    rules: ghost-role-information-space-dragon-summoned-carp-rules
    mindRoles:
    - MindRoleGhostRoleTeamAntagonistFlock
  - type: GhostTakeoverAvailable
  - type: HTN
    rootTask:
      task: DragonCarpCompound
  - type: Flammable
    damage:
      types: {}
  - type: Temperature
    heatDamageThreshold: 1200
  - type: MeleeWeapon
    damage:
      types:
        Slash: 10
        Piercing: 10
```